CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+Please see [doc/docs/project/CHANGELOG.md
```

Comment:
I was just thinking that if you did eventually host the docs somewhere like readthedocs.org, it would also be nice to have the rest of the documentation there too, but still have permanent pointers to the "hosted" docs in the repo root as well. I thought maybe this could satisfy all potential scenarios.